pymrcpp/pyFundamentalTypes.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ *  \date Mar 08, 2018
+ *  \author Magnar Bjørgve <magnar.bjorgve@uit.no> \n
+ *          Hylleraas Centre for Quantum Molecular Sciences \n
+ *          UiT - The Arctic University of Norway
+ */
+
+#include "pybind11/pybind11.h"
+#include "pybind11/numpy.h"
+#include "pybind11/eigen.h"
+#include "pybind11/functional.h"
+
+#include "trees/BoundingBox.h"
+#include "trees/MultiResolutionAnalysis.h"
+#include "trees/MWTree.h"
+#include "trees/FunctionTree.h"
+#include "trees/FunctionTreeVector.h"
+#include "PyBoundingBox.h"
+
+using namespace mrcpp;
+namespace py = pybind11;
+
+
+template<int D>
```

Comment:
Mmmmm, can't the templating be moved down to the bindings for the types themselves, which are true C++ `template`-s? You can write a lambda function to get the name of the class and I would keep the angle brackets notation there.